libraries/core-react/src/Popover/Popover.tsx

```diff
@@ -15,35 +13,59 @@ const Anchor = styled.div`
     outline: none;
   }
 `
+type PopoverChild = {
+  type?: { displayName?: string }
+} & ReactElement
+
+export type Props = {
+  /* Popover placement relative to anchor */
+  placement?:
+    | 'topLeft'
+    | 'top'
+    | 'topRight'
+    | 'rightTop'
+    | 'right'
+    | 'rightBottom'
+    | 'bottomLeft'
+    | 'bottom'
+    | 'bottomRight'
+    | 'leftTop'
+    | 'left'
+    | 'leftBottom'
+  /**  On Close function */
+  onClose?: () => void
+  /**  Open activates <PopoverItem/> */
+  open?: boolean
+  children?: PopoverChild | PopoverChild[]
+} & HTMLAttributes<HTMLDivElement>
+
 // Controller Component for PopoverItem
-export const Popover = forwardRef(function Popover(
-  { className, open, children, ...rest },
+export const Popover = forwardRef<HTMLDivElement, Props>(function Popover(
+  { open, children, placement = 'bottom', ...rest },
   ref,
 ) {
   const props = {
     ...rest,
-    className,
+    placement,
     ref,
   }
 
-  const anchorRef = useRef(null)
+  const anchorRef = useRef<HTMLDivElement>(null)
 
-  let anchorElement
+  let anchorElement: PopoverChild
   const childArray = []
   if (Array.isArray(children)) {
```

Comment:
Maybe make a new const for children where its casted to `PopoverChild | PopoverChild[]` for iteration purpose